VideoLocker/assets/config/registration_form.json

```diff
@@ -35,7 +33,6 @@
         "errorMessages": {},
         "placeholder": "",
         "defaultValue": "",
```

Comment:
Were other options discussed for other ways to show this help? e.g. in placeholder text within the fields?